storage/jellyfish_merkle/src/lib.rs

```diff
@@ -89,9 +89,9 @@ pub trait TreeReader {
 }
 
 /// Node batch that will be written into db atomically with other batches.
-pub type NodeBatch = HashMap<NodeKey, Node>;
+pub type NodeBatch = BTreeMap<NodeKey, Node>;
```

Comment:
I thought Rocks would sort this before commit.  Is that not the case?

Reply:
Yes. But it can still introduce overlapping files since one batch can cross memtables.